server/controllers/placementApplications/withdrawalsController.ts

```diff
@@ -21,12 +24,16 @@ export default class WithdrawalsController {
         req.params.id,
       )
 
+      const user = await this.userService.getActingUser(req.user.token)
```

Comment:
We have this value in the session (`req.session.user`) so it would be better if we read from that

Reply:
Ah nice - will update!